src/components/Login/helpers/TokenService.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable require-jsdoc */
+interface IUserToken {
+  accessToken: string;
+  refreshToken: string;
+}
+
+interface IUserInfo {
+  userId: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  isActivated: boolean;
+  tokens: IUserToken;
+}
+class UserTokenService {
+  getUserInfo(): IUserInfo | undefined {
+    if (typeof window !== "undefined") {
```

Comment:
Why need to judge "window" not to be "undefined"? all of browsers have window object

Reply:
Removed unnecessary conditional statements.

Reply:
Just a reminder, we may need this window condition back later when we use SSR, as some of the codes may not run on the browser, but it is totally good for now. Just saying.